src/index.js

```diff
@@ -16,6 +16,18 @@ const pluginName = 'mini-css-extract-plugin';
 const REGEXP_CHUNKHASH = /\[chunkhash(?::(\d+))?\]/i;
 const REGEXP_CONTENTHASH = /\[contenthash(?::(\d+))?\]/i;
 const REGEXP_NAME = /\[name\]/i;
+const REGEXP_COMMENTS = /\/\*[\s\S]*?\*\/|([^\\:]|^)\/\/.*$/gm;
+
+const moduleHasNoSource = (module) => {
+  // eslint-disable-next-line no-underscore-dangle
+  if (!module._source || !module._source._value) return true;
+
+  return (
+    // strip all comments
+    // eslint-disable-next-line no-underscore-dangle
+    module._source._value.replace(REGEXP_COMMENTS, '').trim().length === 0
```

Comment:
`_*` options is private and can be removed in future or between versions, we need search way how we can do this without using private properties

Reply:
Fixed